### DISCUSSION TITLE
blog/hosting-aspnet-core-with-smarteraspnet/

### DISCUSSION THREAD

**Discussion by giscus** [codewithmukesh/discussions#41] [General]
# blog/hosting-aspnet-core-with-smarteraspnet/

In this article, let's learn about Deploying / Hosting ASP.NET Core with SmarterASP.NET, a Superior Hosting Service for ASP.NET Core 3.x. We will also go step-by-step on deploying an actual ASP.NET Core 3.1 Application directly from Visual Studio 2019 IDE to SmarterASP.NET Server via WebDeploy.

https://codewithmukesh.com/blog/hosting-aspnet-core-with-smarteraspnet/

**Comment by Pxncxke** [comment]
Hi Mukesh, really nice article, exactly what I was looking for.
Question, do you still have any of does discount codes?
Thanks!
